resources/js/Pages/Contact/Create.js

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import Authenticated from '@/Layouts/Authenticated';
+import {Head, useForm} from '@inertiajs/inertia-react';
+import Button from "@/Components/Button";
+import ValidationErrors from "@/Components/ValidationErrors";
+import {FormControlLabel, TextField} from "@mui/material";
+import Checkbox from '@mui/material/Checkbox';
+
+export default function Create(props) {
+    const onHandleChange = (event) => {
+        setData(event.target.name, event.target.type === 'checkbox' ? event.target.checked : event.target.value);
+    };
+
+    const submit = (e) => {
+        e.preventDefault();
+
+        post(route('contact.store'));
+    };
+
+    const { post, processing, errors, data, setData  } = useForm({
+        name: '',
+        city: '',
+        phone: '',
+        age: '',
+        country: '',
+        employee: false,
+    });
+
+    return (
+        <Authenticated
+            auth={props.auth}
+            errors={props.errors}
+            header={<h2 className="font-semibold text-xl text-gray-800 leading-tight">Create contact</h2>}
+        >
+            <ValidationErrors errors={errors} />
+                <Head title="Create" />
+                    <div className="max-w-2xl w-full mx-auto my-8 flex flex-col sm:justify-center items-center pt-6 sm:pt-0 bg-gray-100 ">
+                        <form onSubmit={submit} className="bg-white px-5 py-3">
+                            <div>
```

Comment:
It is suggested to use white space before the element:

```suggestion
                            <div className="mb-4">
```